pkg/users/local.js

```diff
@@ -87,26 +89,38 @@
     if (groupsExtraInfo.length == 0 || accountsInfo.length == 0) {
         return <EmptyStatePanel loading />;
     } else if (path.length === 0) {
-        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groupsExtraInfo || []} />;
-    } else {
+        return (
+            <AccountsMain
+                accountsInfo={accountsInfo}
+                current_user={current_user_info && current_user_info.name}
+                groups={groupsExtraInfo || []}
+                isGroupsExpanded={isGroupsExpanded}
+                setIsGroupsExpanded={setIsGroupsExpanded}
+            />
+        );
+    } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo} shadow={shadow || []}
                             current_user={current_user_info && current_user_info.name} user={path[0]} />
         );
-    }
+    } else if (path.length === 2 && path[0] === "group" && groupsExtraInfo.length > 0 && accountsInfo.length > 0) {
+        const current_group = groupsExtraInfo.find(g => g.name == path[1]);
+        if (current_group) {
+            return (
+                <GroupDetails accounts={accountsInfo} groups={groupsExtraInfo} group={path[1]} />
+            );
+        }
+    } else return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18136-20230202-104351-18903f6b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)